src/migration/tasks/FixChargingStationSiteIDsTask.ts

```diff
@@ -0,0 +1,54 @@
+import Constants from '../../utils/Constants';
+import Logging from '../../utils/Logging';
+import { ServerAction } from '../../types/Server';
+import Tenant from '../../types/Tenant';
+import TenantMigrationTask from '../TenantMigrationTask';
+import Utils from '../../utils/Utils';
+import global from '../../types/GlobalType';
+
+const MODULE_NAME = 'FixChargingStationSiteIDsTask';
+
+export default class FixChargingStationSiteIDsTask extends TenantMigrationTask {
```

Comment:
No need this task.
Why not fixing it in FixChargingStationCompanyIDsTask?

Reply:
True, now it's updated in the same task